### PR TITLE
Add cleanup script.

### PR DESCRIPTION
**Pull request by glogiotatidis**

### CODE REVIEW

File: cleanup.sh [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+            DRY_RUN=false
+            ;;
+    esac
+done
+
+for region in ${REGIONS};
+do
+    echo " ➤ Scanning region ${region}"
+    TO_REMOVE=$(
+        aws ec2 describe-snapshots --owner-ids="${OWNER}" --region="${region}" | \
```
Comment (bookshelfdave) [inline cleanup.sh:24]: do you think there is any possibility that `CUTOFF_DATE` could ever be null/empy/invalid and delete more than we expect?

Comment (glogiotatidis) [reply cleanup.sh:24]: I don't think we can reach this point.

**Comment by glogiotatidis** [comment]
Updated to skip in-use snapshots and to send summary to slack.

Used it to remove older than 365 days snapshots locally and it worked fine.

**Comment by glogiotatidis** [comment]
Re Volumes and AMIs: We only create volumes and AMIs manually and that's why we only have a handful which we can de-register and manage without automation. 

An exception to this is the GitLab Runner AMIs but in this case Packer is configured to remove old AMIs on every new build.

**Review by bookshelfdave — APPROVED** [review]
:+1:

**Pull request merged**